docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -86,6 +86,17 @@ Valid options for `[Container]` are listed below:
 | Exec=/usr/bin/command            | Command after image specification - /usr/bin/command   |
 | ExposeHostPort=50-59             | --expose 50-59                         |
 | Group=1234                       | --user UID:1234                        |
+| HealthCmd="/usr/bin/command"     | --health-cmd="/usr/bin/command"        |
+| HealthInterval=2m                | --health-interval=2m                   |
+| HealthOnFailure=restart          | --health-on-failure=restart            |
```

Comment:
```suggestion
| HealthOnFailure=kill          | --health-on-failure=kill            |
```

The "restart" action doesn't work well in systemd, so we shouldn't encourage users to try it.